app/javascript/app/components/definition-list/definition-list-styles.scss

```diff
@@ -2,11 +2,8 @@
 @import '~styles/layout.scss';
 
 .definitionCompare {
-  @include columns((3, 3, 3, 3));
-  @include xy-gutters(40px, $gutter-position: ('top', 'bottom'), $gutter-type: padding);
-  @include xy-gutters(60px);
-
-  margin: auto;
+  display: grid;
```

Comment:
This last change is sadly breaking the IE layout. IE doesn't like grid. That is why we are using foundation. You will have to add some ms-grid attributes but I recommend reverting this change for now until we have a complete solution for small screens and mobile

![image](https://user-images.githubusercontent.com/9701591/80704641-3c2dcd80-8ae5-11ea-8445-76b3988b5dd7.png)